Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -406,6 +428,33 @@ private void CollectionThreadWorker(object state)
                 Thread.Sleep(timeLeftUntilNextTick > TimeSpan.Zero ? timeLeftUntilNextTick : TimeSpan.Zero);
             }
         }
+        
+
+        private void InitializeCollectionThread()
+        {
+            try
+            {
+                this.topCpuCollector.Initialize();
```

Comment:
No, Initialize() doesn't start a thread. This code is already executed within the top CPU collection thread.